Sources/TootSDK/Models/Instance.swift

```diff
@@ -219,9 +219,12 @@ extension Instance {
         if version.lowercased().contains("akkoma") {
             return .akkoma
         }
-        if version.lowercased().contains("firefish") {
+        if version.lowercased().contains("firefish") || version.lowercased().contains("iceshrimp") {
```

Comment:
With the additional logics, would it be possible to add some unit tests around the new flavour detections?